CoEpi/ui/symptomReport/FollowUp/SymptomStartDaysViewModel.swift

```diff
@@ -9,9 +9,19 @@ class SymptomStartDaysViewModel {
     let title = L10n.Ux.Symptomsdays.heading
 
     let setActivityIndicatorVisible: Driver<Bool>
+    
+    private var daysIsEmpty: BehaviorRelay<Bool>
+    let submitButtonEnabled: Driver<Bool>
 
     init(symptomFlowManager: SymptomFlowManager) {
         self.symptomFlowManager = symptomFlowManager
+        
+        daysIsEmpty = BehaviorRelay<Bool>(value: true)
```

Comment:
@i-schuetz is this a good way to bind the changes in the input text box to the submit button? I tried to go off of how other things were structured.

Reply:
Yes 👍

Reply:
Great, I'll go and implement this in the other views with this pattern.